qiling/os/windows/dlls/kernel32/fileapi.py

```diff
@@ -205,3 +217,19 @@ def hook_CreateFileA(ql, address, params):
 def hook_CreateFileW(ql, address, params):
     ret = _CreateFile(ql, address, params, "CreateFileW")
     return ret
+
+
+# DWORD GetTempPathW(
+#   DWORD  nBufferLength,
+#   LPWSTR lpBuffer
+# );
+@winapi(cc=STDCALL, params={
+    "nBufferLength": DWORD,
+    "lpBuffer": POINTER
+})
+def hook_GetTempPathW(ql, address, params):
+    # TODO not sure if the string should end with \ and have a 00
+    temp = "C:\TEMP"
```

Comment:
Will it be better C:\Windows\Temp ?

Reply:
it should also create rootfs\Windows\Temp ?

Reply:
You are right, it should!